javascript/javascript3/week1/giphyApi/style.css

```diff
@@ -0,0 +1,47 @@
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background: azure;
+}
+
+main * {
+    margin-bottom: 1em;
+}
+.giphy-h1 {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS',
+        sans-serif;
+    font-style: oblique;
+    font-size: 5rem;
+    color: aqua;
+    padding: 0;
+
+    margin-bottom: 3rem;
+}
+input[type='text'] {
+    border: none;
+    outline: none;
+    border-radius: 24px;
+    padding: 1em;
+    background: #0a0a0a;
+    color: white;
+    font-size: 1rem;
+}
+.search-gif-button {
+    background-color: #4caf50; /* Green */
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 1em;
+    cursor: pointer;
+}
+.search-limit {
+    width: 82px;
+}
+img {
```

Comment:
Here again, img tag is a generalized one. Maybe you would have one more img tag on the same app later but you'd want a different style for that. Doesn't matter much, but still it's a matter of good coding habit. No need to change it here, but remember for later.